app/assets/javascripts/discourse/controllers/composer.js.es6

```diff
@@ -719,15 +715,15 @@ export default Ember.Controller.extend({
         this.close();
 
         const currentUser = Discourse.User.current();
```

Comment:
While you're in there, is this not available as `this.currentUser`?

Reply:
Oh yes good catch, I will add this to my list of things to think of when refactoring a file :+1: